src/Schedule.cpp

```diff
@@ -233,7 +233,7 @@ typedef std::map<FunctionPtr, FunctionPtr> DeepCopyMap;
 struct FuncScheduleContents {
     mutable RefCount ref_count;
 
-    LoopLevel store_level, compute_level;
+    LoopLevel store_level, compute_level, hoist_storage_level;
```

Comment:
This probably requires serialization changes

Reply:
I changed the code for serialization and hopefully not missed anything.